apps/site/src/views/Home/sections/FAQ/FAQ.tsx

```diff
@@ -1,11 +1,63 @@
+import { useEffect, useState } from "react";
 import Container from "react-bootstrap/Container";
+import Accordion from "react-bootstrap/Accordion";
+import styles from "./FAQ.module.scss";
+
+const santiy_url: string =
+	"https://1smqaeyk.api.sanity.io/v2021-10-21/data/query/production?query=*%5B_type+%3D%3D+%22faqs%22%5D+&perspective=previewDrafts";
+
+interface sanity_response {
+	result: {
+		faqs: faq_type[];
+	}[];
+}
+interface faq_type {
+	question: string;
+	answer: string;
+}
+
+export default function FAQ() {
+	const [faq, setFAQ] = useState<faq_type[] | []>([]);
+
+	useEffect(() => {
+		fetch(santiy_url)
+			.then((response) => {
+				response
+					.json()
+					.then((data: sanity_response) => {
+						setFAQ(data.result[0]["faqs"]);
+					})
+					.catch((e: Error) => {
+						console.log(e);
+					});
+			})
+			.catch((e: Error) => {
+				console.log(e);
+			});
+	}, []);
 
-function FAQ() {
 	return (
 		<Container as="section">
-			<h2>FAQ</h2>
+			<div className={styles.faq}>
+				<div className={styles.vertical_line} />
+				<h1 className={styles.title}>FAQ</h1>
+				<Accordion className={styles.accordion} alwaysOpen>
+					{faq.map((q, index) => (
+						<Accordion.Item
+							key={index}
+							className={styles["accordion-border"]}
+							eventKey={`${index}`}
+						>
+							<Accordion.Header as="h3" className={styles.body}>
+								{q["question"]}
+							</Accordion.Header>
```

Comment:
Suggestion: to better differentiate the question from the answer, the header could be bolded.